publisher-sdk/src/main/java/com/criteo/publisher/util/BuildConfigWrapper.java

```diff
@@ -24,6 +25,10 @@
  */
 public class BuildConfigWrapper {
 
+  public int getAndroidSdkVersion() {
```

Comment:
I've put in the `BuildConfigWrapper` because this class is a pure wrapper without logic, without any dependencies.
So it can be instantiated and used even before the SDK init (used to choice if SDK is a dummy or not).

`DeviceUtil` needs the `Context`, and then cannot be used before the SDK init.

I can add a new wrapper class but I'm afraid that it will be just more technical debt. Or we can rename the `BuildConfigWrapper` to something like `ConstantWrapper`

Reply:
Finally I skipped this commit. By removing the READ_PHONE_STATE permission, lot of things are simplified and this is not necessary anymore.